doc/man3/EVP_PKEY_verify.pod

```diff
@@ -50,31 +50,31 @@ I<ctx> for verifying an unlimited size message using the algorithm given by
 I<algo> and the key given through L<EVP_PKEY_CTX_new(3)> or
 L<EVP_PKEY_CTX_new_from_pkey(3)>.
 Passing the message is supported both in a one-shot fashion using
-EVP_PKEY_verify(), and through the combination of EVP_PKEY_verify_update() and
-EVP_PKEY_verify_final().
+EVP_PKEY_verify(), and through the combination of
+EVP_PKEY_verify_message_update() and EVP_PKEY_verify_message_final().
 This function enables using algorithms that can process input of arbitrary
 length, such as ED25519, RSA-SHA256 and similar.
 
 EVP_PKEY_CTX_set_signature() specifies the I<siglen> bytes long signature
-I<sig> to be verified against by EVP_PKEY_verify_final().
-It I<must> be used together with EVP_PKEY_verify_update() and
-EVP_PKEY_verify_final().
+I<sig> to be verified against by EVP_PKEY_verify_message_final().
+It I<must> be used together with EVP_PKEY_verify_message_update() and
```

Comment:
Not sure if these ones should be changed..

Reply:
There is no function called EVP_PKEY_verify_update() or EVP_PKEY_verify_final(), why wouldn't you want to fix this issue ?